### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by Bobsans** [klinkin/pyunisend#6]

**Comment by Bobsans** [comment]
```
File "\home\xxx\virtual-env-test\lib\site-packages\pyunisend\pyunisend.py", line 5, in <module>
    import urllib2
ImportError: No module named 'urllib2'
```

**Comment by qari** [comment]
Please use FORK [https://github.com/qari/pyunisend](https://github.com/qari/pyunisend)

**Comment by klinkin** [comment]
Fix by #8
